src/test/scala/GoTests.scala

```diff
@@ -1 +1,102 @@
 package game
+
+import org.scalatest.FunSuite
+
+class GoTests extends FunSuite {
+  val players: Stream[Player] = Stream.continually(List(Black, White)).flatten
+
+  def show(position: Position[Point5]): String =
+    (1 to 5).map { y =>
+      (1 to 5).map { x =>
+        position.at(Point5(x, y)) match {
+          case Empty => "."
+          case Stone(Black) => "x"
+          case Stone(White) => "o"
+        }
+      }.mkString(" ")
+    }.mkString("|")
+
+  test("5x5 capture") {
+    val positions = Go.playAll(Seq(Point5(1, 2), Point5(1, 1), Point5(2, 1)).map(Move.apply))
+    assert(show(positions.right.get.head) ==
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2586331318/issues/source?bid=3301982&fileBranchId=3302581#l21)